ssl/ssl_cipher.cc

```diff
@@ -1834,7 +1834,15 @@ bool tls_print_all_supported_cipher_suites(bool use_openssl_name) {
 }
 
 const char *SSL_CIPHER_get_version(const SSL_CIPHER *cipher) {
-  return "TLSv1/SSLv3";
+  switch (SSL_CIPHER_get_min_version(cipher)) {
+    case TLS1_2_VERSION:
+    case DTLS1_2_VERSION:
```

Comment:
should not have DTLSv1.2?

Reply:
other `SSL_CIPHER_*` functions seem to treat TLS/DTLS as the "same" with respect to ciphers (see [here](https://github.com/aws/aws-lc/blob/main/ssl/ssl_cipher.cc#L1587)), and appear to use "TLS" for labels.